app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/Toggle.kt

```diff
@@ -211,33 +210,45 @@ private fun Toggle(enabled: Boolean, onClick: (Boolean) -> Unit, modifier: Modif
       },
     )
   Crossfade(
-    // todo: looks fine without animating too, bc the is this container color change anyway
     targetState = enabled,
     animationSpec = tween(AnimationTokens().pulsatingAnimationDuration),
+    modifier = modifier
   ) { animatedEnabled ->
     Box(modifierNoIndication) {
```

Comment:
Could we inline this `modifierNoIndication` so that the val does not exist now after the latest changes?